dist/s_string.ok

```diff
@@ -1363,6 +1363,7 @@ reentrant
 refp
 regionp
 reinitialization
+reinstantiated
```

Comment:
This change also may not needed for 6.0

Reply:
Good catch

Reply:
bf6055567d1725943a2cd5885e4ae977a3ea3614